src/bindings/js/node/include/async_reader.hpp

```diff
@@ -6,6 +6,7 @@
 #include <napi.h>
 
 #include "openvino/runtime/core.hpp"
+#include "openvino/core/model_util.hpp"
```

Comment:
If `model_util.hpp` is not used here in header it can be moved to implementation file to limit scope

Reply:
Thanks, it has been fixed!